src/lib.rs

```diff
@@ -133,6 +132,23 @@ pub fn local_ip() -> Result<IpAddr, Error> {
             .ok_or(Error::LocalIpAddressNotFound)
     }
 
+    #[cfg(any(
+        target_os = "ios"
+    ))]
```

Comment:
I think this can be condensed into:

```suggestion
    #[cfg(target_os = "ios")]
```